src/main.rs

```diff
@@ -1,11 +1,14 @@
+// Set SDK_KEY to your LaunchDarkly SDK key.
+// Set FEATURE_FLAG_KEY to the feature flag key you want to evaluate.
+
```

Comment:
@mmrj I haven't figured out how to set the key and flag variables correctly. I see [here](https://www.tutorialspoint.com/rust/rust_variables.htm#:~:text=Simply%20put%2C%20a%20variable%20helps,be%20performed%20on%20the%20variable.) that they should probably be in the form of `let variable_name = value;` or `let variable_name:dataType = value;`, but my attempt to add them resulted in errors when running `cargo run -q`.
I tried `let SDK_KEY = sdk-abc;` (except using my real SDK key) and got the error `error: expected item, found keyword `let`. I tried a few other permutations with no success. Thoughts?

Reply:
Based on the old / existing version of the README, these are set up to be environment variables. Environment variables are variables that you define in your terminal, outside of this app, rather than in main.rs. (Line 7/10 that starts `let sdk_key = std::env::var("` is saying to set main.rs's variable`sdk_key` to the value of an `env` variable called `SDK_KEY`.)

For your own testing, open a terminal in the `hello-rust` directory and try `export LAUNCHDARKLY_SDK_KEY=xx` and `export FEATURE_FLAG_KEY=xx` to set the environment variables before you try and run this.

For the updates, if the above works we keep this using environment variables, then new lines 1-2 aren't needed. You could take language from these and add to the readme build instructions if you want.

Alternatively, if you do want to try and switch these from environment variables to variables declared within `main.rs`, I _think_ it'd be:

```
env_logger::init();
let sdk_key = 'sdk-abc';
let feature_flag_key = 'my-boolean-flag';
```

because `sdk_key` is the name of the variable that's used later in `main.rs` (as opposed to `SDK_KEY`, which is the name of the environment variable). I'm also pretty sure the quotes are required. But I haven't tested this.

Reply:
(All that said, I don't have any particular insight into why this is app is set up to use environment variables vs. in-app variables.)

Reply:
Thank you Molly! I had tried setting the environment variables in the terminal, but I didn't use the "" around the values, which turned out to be the problem. `export LAUNCHDARKLY_SDK_KEY="xx"` worked.